src/appliance-onboarding-script/appliance_setup/pkgs/_appliance_setup.py

```diff
@@ -203,15 +205,19 @@ def _deploy_and_create_appliance(self) -> str:
         with TempChangeDir(self._temp_dir):
             config = self._config
             apiserver_address = config['applianceControlPlaneIpAddress']
-            if is_api_server_reachable and not confirm_prompt(f'An ApiServer is already reachable on endpoint {apiserver_address}. Deployment will be skipped. Do you want to continue?'):
-                raise ProgramExit('User chose to exit the program.')
+            # Removing confirm_prompts for automated testing 
+            # TODO Check what needs to be done in case there is a reachable ApiServer
+            # For now, we consider the Api server to be correctly configured and don't deploy a new one.
+            #if is_api_server_reachable and not confirm_prompt(f'An ApiServer is already reachable on endpoint {apiserver_address}. Deployment will be skipped. Do you want to continue?'):
```

Comment:
We are removing the prompt. Can we get an ack from Hetal if this experience change is alright? Let's list all places that will have the experience change

Reply:
Also, can we add a log as information of the selection being made?

Reply:
Retained current experience and switched experience based on isAutomated for automation test.